### ISSUE TITLE
Does not work with payment Blockchain

### ISSUE THREAD

**Issue by Hubbardcoin** [ITPrism/CrowdFunding#180]
0 - Unable to decode JSON response from Blockchain:

You may not be able to visit this page because of:

an out-of-date bookmark/favourite
a search engine that has an out-of-date listing for this site
a mistyped address
you have no access to this page
The requested resource was not found.
An error has occurred while processing your request.

**Comment by ITPrism** [comment]
Thank you for the post!

You are right. Blockchain changed their API and the current plugin is not compatible with the new API.
I am waiting to receive API key from them to do the plugin with the new API.

If you would like, you can also try to get API keys. If they give you, you can share it with me and I will do the plugin to work with the new API.

**Comment by Hubbardcoin** [comment]
Yes, I'll try to get the key, and be sure to share with you the.
